stetho/src/main/java/com/facebook/stetho/inspector/elements/android/AndroidDOMProvider.java

```diff
@@ -48,6 +50,24 @@
   private final InspectModeHandler mInspectModeHandler;
   private Listener mListener;
 
+  // We don't yet have an an implementation for reliably detecting fine-grained changes in the
+  // View tree. So, for now at least, we have a timer that runs every so often and just reports
+  // that we changed. Our listener will then read the entire DOM from us and transmit the changes to
+  // Chrome. Detecting, reporting, and traversing fine-grained changes is a future work item.
```

Comment:
actually these docs belong on the 1000 constant.  The constants stand out more IMO and are better places to document unexpected behaviours like this